kem/README.md

```diff
@@ -0,0 +1,61 @@
+# RustCrypto: Key Encapsulation Mechanisms
+
+[![crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+![Apache2/MIT licensed][license-image]
+![Rust Version][rustc-image]
+[![Project Chat][chat-image]][chat-link]
+[![Build Status][build-image]][build-link]
+
+This crate contains traits which provide generic, object-safe APIs for
+establishing shared secrets using [a key encapsulation mechanism][1].
+
+[Documentation][docs-link]
+
+## Minimum Supported Rust Version
+
+Rust **1.41** or higher.
```

Comment:
Not sure what actual MSRV you have in mind, but this doesn't match what's in lib.rs, which is 1.56.

If you make this a 2021 edition crate, you can unify the toplevel rustdoc with what's in README.md, as it were.